includes/class-wc-payments-order-service.php

```diff
@@ -83,6 +83,13 @@ class WC_Payments_Order_Service {
 	 */
 	const WCPAY_REFUND_ID_META_KEY = '_wcpay_refund_id';
 
+	/**
+	 * Meta key used to store WCPay refund transaction id.
+	 *
+	 * @const string
+	 */
+	const WCPAY_TRANSACTION_ID_META_KEY = '_refund_transaction_id';
```

Comment:
IMO, it should be more precisely named as `WCPAY_REFUND_TRANSACTION_ID_META_KEY`.

Also, I noticed that all constants that start with `WCPAY_` are linked to key names that also start with `_wcpay_`. Going by that logic, it should be:

```suggestion
	const WCPAY_REFUND_TRANSACTION_ID_META_KEY = '_wcpay_refund_transaction_id';
```

Reply:
changed in https://github.com/Automattic/woocommerce-payments/pull/7932/commits/f05ac4211d57c00f76794c5b13fdddb6e14b9dc6 . thx